content/concepts/asset-manager.md

```diff
@@ -2075,19 +2105,19 @@ Note that the `channel` property is set to `null` in this case.
 
 #### Asset values of a scopable attribute
 
-The `end_of_use_date` attribute is scopable but not localizable, so it can hold several data values, up to one per channel.
+The `model_height` attribute is scopable but not localizable, so it can hold several data values, up to one per channel.
 ```json
 {
-  "end_of_use_date": [
+  "model_height": [
     {
       "locale": null,
       "channel": "ecommerce",
-      "data": "02/03/2021"
+      "data": "1.65m"
     },
     {
       "locale": null,
       "channel": "print",
-      "data": "03/02/2021"
+      "data": "165cm"
```

Comment:
same

Reply:
The previous example illustrated a localizable text value by using a date. We preferred changing the example to avoid confusion with the new date attribute type.